task3/content/js/index.js

```diff
@@ -0,0 +1,122 @@
+const closeBtnInput = document.getElementById('closeBtn');
+const submitBtnInput = document.getElementById('submitBtn');
+const editContainerInput = document.querySelector('.edit-container');
+
+let tableData = [
```

Comment:
The table data variable is not wrong but it is not the best practice, you need only to mention the user data not with the table elements.
`let tableData=[{name: 'user1', cart: '435$'}]` etc...